public/services/utils/helper.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  ChannelItemType,
+  // NotificationItem,
+  // RecipientGroupItemType,
+  // SenderItemType,
+} from '../models/interfaces';
+
+export const configToChannel = (config: any): ChannelItemType => {
+  return {
+    ...config.config,
+    config_id: config.config_id,
+    created_time_ms: config.created_time_ms,
+    last_updated_time_ms: config.last_updated_time_ms,
+  };
+};
+
+export const configListToChannels = (configs: any[]): ChannelItemType[] => {
+  return configs?.map((config) => configToChannel(config)) || [];
+};
+
+// export const configToSender = (config: any): SenderItemType => {
+//   return {
+//     name: config.config.name,
```

Comment:
Can we remove these comments?

Reply:
Will do a clean-up passthrough in a follow-up